iidm/iidm-converter-api/src/main/java/com/powsybl/iidm/import_/Importers.java

```diff
@@ -280,18 +280,36 @@ public static Importer removePostProcessors(Importer importer) {
      * @param dataSource data source
      * @param parameters some properties to configure the import
      * @param computationManager computation manager to use for default post processors
+     * @param config the import configuration
+     * @param reporter the reporter used for functional logs
      * @return the model
      */
-    public static Network importData(ImportersLoader loader, String format, ReadOnlyDataSource dataSource, Properties parameters, ComputationManager computationManager, ImportConfig config) {
+    public static Network importData(ImportersLoader loader, String format, ReadOnlyDataSource dataSource, Properties parameters, ComputationManager computationManager, ImportConfig config, Reporter reporter) {
         Importer importer = getImporter(loader, format, computationManager, config);
         if (importer == null) {
             throw new PowsyblException("Import format " + format + " not supported");
         }
-        return importer.importData(dataSource, NetworkFactory.findDefault(), parameters);
+        if (reporter == Reporter.NO_OP) {
+            return importer.importData(dataSource, NetworkFactory.findDefault(), parameters);
+        } else {
+            return importer.importData(dataSource, NetworkFactory.findDefault(), parameters, reporter);
+        }
```

Comment:
We shouldn't need to do this distinction

Reply:
you are right. fixed.